core/arch/asm_defines.asm

```diff
@@ -196,11 +198,15 @@ ASSUME fs:_DATA @N@\
 #  define PUSH_SEH(reg) push reg @N@ .pushreg reg
 /* Push a volatile register or an immed in prolog: */
 #  define PUSH_NONCALLEE_SEH(reg) push reg @N@ .allocstack 8
+# define ADD_STACK_ALIGNMENT sub REG_XSP, FRAME_ALIGMMENT - ARG_SZ @N@ .allocstack 16
```

Comment:
This is adjusting by 8 bytes so it should be `.allocstack 8`, not 16.

Reply:
I'm completely blind there not knowing the Windows compiler. This doesn't really make sense though does it, you mean you need to tell the Windows compiler how many static bytes to adjust? The computation is dynamic. Happy to believe you that allocstack 8 is correct, unless you want me to research what it means.

Reply:
These directives go into a special section of the binary and are used for frame unwinding at runtime for exception handling.  I have a pointer to this page which has some details: http://msdn2.microsoft.com/en-us/library/tawsa7cb.aspx

The reason allocstack would be 8 is b/c that's how much you're adjusting the stack pointer by, right?  That's what the unwinder cares about.

Reply:
I shouldn't have said "dynamic", it's static in this case. I was wondering how the compiler is using the directive, e.g. if the adjustment *is* dynamic. In any case, set to "8".